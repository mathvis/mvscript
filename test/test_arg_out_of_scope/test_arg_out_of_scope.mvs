func f(a: int) {
	a += 1;
}
a -= 1;
