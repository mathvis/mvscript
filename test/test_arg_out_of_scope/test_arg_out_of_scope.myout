--> test/test_arg_out_of_scope/test_arg_out_of_scope.mvs:4:1
error: Variable "a" was not initialized.
