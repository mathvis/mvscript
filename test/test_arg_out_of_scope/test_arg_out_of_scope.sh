#!/bin/bash
helix test_arg_out_of_scope.mvs test_arg_out_of_scope.out --vsplit
