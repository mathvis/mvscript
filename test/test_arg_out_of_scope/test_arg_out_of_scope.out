mvscript: Variable "a" was not initialized.
CallStack (from HasCallStack):
  error, called at lib/VariableStorage.hs:69:16 in mvscript-0.1.0.0-inplace-mvscript-lib:VariableStorage
